config/routes.rb

```diff
@@ -237,3 +237,8 @@
   end
 end
 # rubocop:enable Metrics/BlockLength
+Rails.application.routes.draw do
```

Comment:
there is already a `Rails.application.routes.draw do` here: https://github.com/CircuitVerse/CircuitVerse/blob/0762999968fc96c632edd6a3fe5beca776b8c416/config/routes.rb#L4, you can add it here